wrappers/python/simtk/openmm/app/charmmparameterset.py

```diff
@@ -100,7 +100,7 @@ def _convert(data, type, msg=''):
         except ValueError:
             raise CharmmFileError('Could not convert %s to %s' % (msg, type))
 
-    def __init__(self, *args):
+    def __init__(self, *args ):
```

Comment:
The constructor should also take `permissive`.  You can do this via `**kwargs` and then popping off the `permissive` keyword (and then to avoid silently passing errors, throw a `TypeError` if the `kwargs` dict is not empty after you pop off `permissive`).